src/extensions/ResizableImage.ts

```diff
@@ -18,6 +20,11 @@ export type ResizableImageOptions = ImageOptions & {
    * hostnames are allowed.
    */
   isAllowedImgSrc(src: string | null): boolean;
+
+  /**
+   * Optional React component to pass in as a child component to ResizableImage.
```

Comment:
minor: perhaps something to the effect of "Optional React component to render inside the ResizableImage container element, as a sibling of the image element", just to be more explicit about where this shows up and how it differs from the `ResizableImageComponent` itself.